datafusion/physical-expr/src/equivalence/ordering.rs

```diff
@@ -74,6 +74,11 @@ impl OrderingEquivalenceClass {
         result
     }
 
+    /// Returns the orderings in this ordering equivalence class.
+    pub fn orderings(&self) -> &[LexOrdering] {
```

Comment:
Needed this to pass the orderings in the next file diff. Let me know if this is not a good way to do it.